scripts/squads.yml

```diff
@@ -0,0 +1,218 @@
+    github: MichalTKwiecien
+    twitter: kwiecien_co
+  - &Olivier
+    name:  Olivier Halligon
+    github: AliSoftware
+    twitter: aligatr
+  - &Patrick
+    name: Patrick Westmeyer
+    github: bh-pwestmeyer
+  - &Rui
+    name: Rui Peres
+    github: RuiAAPeres
+    twitter: peres
+  - &Sam
+    name: Sam Francis
+    github: SamFrancis-Babylon
+  - &Sergey
+    name: Sergey Shulga
+    github: sergdort
+    twitter: SergDort
+  - &Simon
+    name: Simon Cass
+    github: scass91
+    twitter: codercass
+  - &Viorel
+    name: Viorel Mihalache
+    github: viorel15
+    twitter: viorelMO
+  - &Witold
+    name: Witold Skibniewski
+    github: mr-v
+  - &Yuri
+    name: Yuri Karabatov
+    github: karabatov
+    twitter: karabatov
+
+
+################################################################
+# Global roles transverse to the squads
+#
+roles:
+  - name: Engineering Manager
+    members:
+      - *Rui
+  - name: Chapter Leads
+    members:
+      - *Catarina
+      - *David
+  - name: Squad Tech Leads
+    members:
+      - *Michael
+
+
+################################################################
+# List of the Squads and their members
+#
+squads:
+  - name: Booking
+    desc: Face to face appointment booking.
+    members:
+      - *Witold
+  - name: Consultation
+    desc: Flows for all the consultation like prescriptions and video consultation.
+    members:
+      - *Adrian
+      - *Chitra
+  - name: Onboarding and navigation
+    desc: General user experience in the app.
+    members:
+      - *Emese
+      - *Giorgos
+      - *Sergey
+      - *Yuri
+  - name: Native Apps Platform
+    desc: Engineering work like tooling, CI and development processes.
+    members: 
+      - *David
+      - *Ilya
+      - *Inamiy
+      - *Martin
+      - *Olivier
+  - name: GP @ Hand
+    desc: End to end journey for NHS registration.
+    members: 
+      - *James
+  - name: Healthcheck
+    desc: Overview of your health using a 3D body model (avatar).
+    members: 
+      - *Ben
+      - *Julien
+  - name: Monitor
+    desc: Monitoring health metrics like activity, blood, urine, ...
+    members: 
+      - *Anders
+      - *DanielH
+      - *DanielS
+      - *Diego
+      - *Joshua
+  - name: Prescriptions
+    desc: Precriptions functionality.
+    members: 
+      - *Adam
+      - *Konrad
+  - name: Partnerships Squad
+    desc: Updating and maintaining the Telus app.
+    members: 
+      - *Simon
+  - name: SDK
+    desc: Develop and maintain the SDK frameworks.
+    members: 
+      - *Viorel
+  - name: Tenancy & Features
+    desc: To integrate the new feature configurator.
+    members: 
+      - *Anil
+  - name: Test Kits
+    desc: Managing everything related to Babylon do-at-home tests.
+    members: 
+      - *Michal
+  - name: Triage
+    desc: Chatbot functionality.
+    members: 
+      - *Danilo
+      - *Michael
+  - name: US Professional Services
+    desc: Features for app in the US.
+    members: 
+      - *Greg
```

Comment:
```suggestion
      - *Chesley
      - *Greg
```

Reply:
Btw @karabatov any idea who Greg is? We don't have either his GitHub handle nor his twitter and I can't find anyone named Greg on our slack channel...

Reply:
I was under the impression that it's Greg Bryant the engineering manager for US? 🤔

Reply:
Well yes it's supposed to be Greg Bryant: https://github.com/Babylonpartners/ios-playbook/pull/225/files#diff-c190676900d06dacf5d18fb8436459c2R59

But if he's a manager, not sure why he's in this list. No other squad list the Engineering manager (nor the DM nor Android devs nor QA, for that matter, since it's the ios-playbook), so not sure why he's in that list then?

Reply:
Ok we can remove it then.

Reply:
I've no idea :) He's not contributing directly, that's for sure.